src/Identification/Queue/Events/Processing.php

```diff
@@ -64,7 +65,12 @@ public function __construct(JobProcessing $event)
             $job = $this->unserializeToJob($command);
         }
 
-        $tenant = $job->getTenant();
+        if ($event->connectionName == "sync" && !is_null(Tenancy::getTenant())) {
```

Comment:
is `sync` meant to be the name of the connection configuration in the config files?  If so, it'd be much better if we could key off of the driver value of `sync`(https://github.com/laravel/laravel/blob/8.x/config/queue.php#L34) instead of the name since the name could be changed to anything